building_energy_standards_data/database_files/hvac_minimum_requirement_heat_rejection_IECC.json

```diff
@@ -0,0 +1,530 @@
+[
+    {
+        "template": "IECC-2006",
+        "equipment_type": "Open Cooling Tower",
+        "fan_type": "Centrifugal",
+        "test_fluid": null,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_performance_gpm_per_hp": 20,
+        "minimum_performance_btu_per_hr_per_hp": null,
+        "annotation": "From TABLE 503.2.3(11)"
```

Comment:
Let's use the same annotation format as for the 90.1 data, see [here](https://github.com/pnnl/building-energy-standards-data/blob/6aeb8df12549fe156ff758f81897e88ae6065627/building_energy_standards_data/database_files/hvac_minimum_requirement_heat_rejection_90_1.json#L11).